map-frontend/package.json

```diff
@@ -8,14 +8,16 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "axios": "^0.19.2",
     "bootstrap": "4.2.1",
     "bootstrap-select": "1.13.5",
     "jquery": "3.3.1",
     "leaflet": "1.4.0",
     "vue": "2.5.21",
     "vue-resource": "1.5.1",
     "vue-router": "3.0.1",
-    "vue-search-select": "2.8.2"
+    "vue-search-select": "2.8.2",
+    "vuex": "^3.1.3"
```

Comment:
Please set fixed dependency version (`"vuex": "3.1.3"`)

Reply:
Fixed in f7d55ab